include/coreir/ir/dynamic_bit_vector.h

```diff
@@ -503,13 +501,6 @@ namespace bsim {
 
       return str;
     }
-
-    bool is_binary() {
-      for (int i = 0; i < bitLength(); i++) {
-        if (!get(i).is_binary()) return false;
-      }
-      return true;
-    }
```

Comment:
Is this function being removed intentionally? This will break pycoreir and the c-api. Is there another way we should be doing this?

Reply:
I moved is_binary in the file. It is now [here](https://github.com/dillonhuff/coreir/blob/324ada898218155ef22e568632046e1d5882f529/include/coreir/ir/dynamic_bit_vector.h#L486)

Reply:
Also it is now ```const``` as it should be.

Reply:
got it